### DISCUSSION TITLE
Agregar la opción de seleccionar la coordenada con valores exactos.

### DISCUSSION THREAD

**Discussion by MaCuur** [grupoioa/gis_agm_series#47] [Ideas]
suponiendo que se está planeando utilizar un punto en específico, ya sin necesidad de recurrir a atinarle al punto, generar la opción de escribir como tal la coordenada para que se desplieguen los parámetros referidos a tal punto

**Comment by ma-robles** [comment]
muy buena idea, lo que estaba pensando es que las coordenadas que aparecen junto con el nombre del punto (en la selección de variables), sea editable y se sincronice con el marcador (que aun no existe) en el mapa

**Reply by MaCuur** [reply]
si, me agrada la idea
